### ISSUE TITLE
Dell XPS 9320: Mic-Mute LED not working

### ISSUE THREAD

**Issue by heftig** [alsa-project/alsa-ucm-conf#251]
The mic-mute LED on my XPS 9320 does not work. Reading `/sys/class/sound/ctl-led/mic/card0/list` outputs `52 18`.

Control 18 is the expected `rt714 FU02 Capture Switch` set in `ucm2/codecs/rt715-sdca/init.conf`. Control 52 is:

```
numid=52,iface=MIXER,name='PGA5.0 5 Master Capture Switch'
  ; type=BOOLEAN,access=rw------,values=2
  : values=on,on
```

In this state, the LED does turn on when both switches are off, but muting via PipeWire only toggles control 18.

Detaching the LED from `52` (so that only `18` remains) makes it work properly.

I've tried finding out how 52 gets attached, but couldn't find anything. Is this done in-kernel?

Linux 6.1
ALSA 1.2.8
Arch Linux

**Comment by perexg** [comment]
Already fixed in 79a8ec44d3dcf097f4a4492c506cbcf338324175 .

**Comment by heftig** [comment]
Is there a release scheduled which contains the fix?
